index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```

Comment:
Please keep the script tags at the very bottom of the document so that you dodge any future errors that might
happen when you do this, as the browser first reads your markup and then executes the scripts that are targeting
your HTML. Kindly check this [article](https://medium.com/m/global-identity?redirectUrl=https%3A%2F%2Flevelup.gitconnected.com%2Fall-about-script-87fea475b976#:~:text=When%20you%20place%20your%20JavaScript,speed%20up%20website%20response%20time.) on how should you place your script tags. 😊✌🏻